setup.py

```diff
@@ -17,6 +17,7 @@
         "pycryptodomex",
         "requests",
         "pyyaml",
+        "setuptools"
```

Comment:
```suggestion
        "setuptools",
```
@aorumbayev Lint is failing

Reply:
@neilcampbell fixed